py3status/docstrings.py

```diff
@@ -111,7 +111,7 @@ def create_readme(data):
 re_from_item = re.compile('^\s+-')
 re_from_data = re.compile('^@(author|license|source)($|[ \t])')
 re_from_tag = re.compile('((`[^`]*`)|[<>&])')
-re_from_defaults = re.compile('(\(default.*\))')
+re_from_defaults = re.compile('(\(default.*\))$')
```

Comment:
Let's keep it simple do you want to just allow whitespace between the `)` and the end of the line and then we can think about a proper parser another day